en/peripherals/gripper.md

```diff
@@ -0,0 +1,106 @@
+# Gripper Peripherals
+
+![High-load gripper example](../../assets/hardware/grippers/highload_gripper_example.jpg)
+
+Grippers are a commonly used mechanism for UAV delivery applications. There are multiple types of interfaces (e.g. PWM, CAN) for different products in the market.
+
+## Supported Gripper Types
+
+PX4 supports the following Gripper types, check the individual page to learn how to integrate them.
+
+- [Servo Gripper with PWM](gripper_servo.md)
+
+:::note
+The [Roboclaw driver](../modules/modules_driver.md#roboclaw) isn't supported as a Gripper yet. However in the future it may get supported as a Gripper.
+:::
+
+## Gripper Actuator Mapping
+
+To enable the output of the Gripper, set the Function of the output port the gripper is connected to as `Gripper`.
+
+:::note
+Make sure to set the correct PWM frequency (usually commercial servo / grippers only support 50Hz)! If you mis-configure the frequency you risk damaging the gripper.
+:::
+
+![Gripper output mapping](../../assets/config/gripper/qgc_gripper_output_setup.png)
+
+## Gripper Action Joystick Button Mapping in QGC
+
+QGC provides the Button mapping interface, which allows to map delivery mechanism actions to specific buttons of the [Joystick](../config/joystick.md) connected to QGC.
+
+### Buttons Mapping
+
+Navigate by clicking **"QGC Logo (upper-left)" > "Vehicle Setup" > "Joystick" > "Button Assignment"** tab. There are Gripper Open and Gripper Close actions available to map to a button.
+
+![Gripper action mapping](../../assets/config/gripper/qgc_gripper_actions_joystick.png)
+
+You can test the actions by clicking on the buttons each and checking Gripper's movements. If the gripper doesn't move as expected, please make sure that the Gripper integration was set up properly.
+
+If you press "Grab" button while the gripper is opening, it will automatically abort releasing behavior and go to the Close position, effectively cancelling the release command.
+
+:::note
+Note that currently in PX4 the Gripper isn't supported as a standalone driver, but is instead tied to the [payload delivery feature](../advanced_features/package_delivery.md).
+
+Therefore unless you [configure the gripper for package delivery](#setting-up-gripper-for-package-delivery), gripper will not be functioning!
```

Comment:
To confirm, the package delivery setup is what links the mavlink gripper command and the gripper actuator output right?

So if you don't enable the payload delivery feature the system might get the mavlink command to grip/ungrip but won't pass that through to the servo?

Reply:
Related to the [comment above](https://github.com/PX4/PX4-user_guide/pull/2072#discussion_r1020336768), gripper's functionality shouldn't be dependent on the package delivery feature ultimately.

And so for now, package delivery setup is a necessary evil for getting gripper to work, but that will be resolved in the future.

Reply:
Yes, I get that. I was hoping you would confirm the "detail". But I think I'm correct.